### PR TITLE
Pythonify the code

### PR DESCRIPTION
**Pull request by Montspy**
Refactoring the code to make use of python utilities.

### CODE REVIEW

**Comment by Montspy** [comment]
`generate.py` ready for review
`metadata.py` WIP

**Comment by Montspy** [comment]
`metadata.py` refactored
@sk33z3r I'd love a second pair of eye on the generated metadata.json
```json
{
    "name": "Collection Name #001",
    "image": "ipfs://QmQtoatsCID/collection_name_001.png",
    "animation_url": "ipfs://QmQtoatsCID/collection_name_001.png",
    "description": "Collection Name #1",
    "royalty_percentage": null,
    "tokenId": 1,
    "artist": null,
    "minter": null,
    "attributes": [
        {
            "trait_type": "BG",
            "value": "BG 01"
        },
        {
            "trait_type": "Guy",
            "value": "Guy 02"
        },
        {
            "trait_type": "Eyes",
            "value": "Eyes 03"
        }
    ],
    "properties": {
        "BG": "BG 01",
        "Guy": "Guy 02",
        "Eyes": "Eyes 03"
    }
}
```

**Comment by Montspy** [comment]
For merge, use incoming changes as this PR includes the same fix as commit 0435287

**Review by sk33z3r — CHANGES_REQUESTED** [review]
Thinking on this, I know that some testers actually wanted a transparent background as a trait that might occur. However, this may not be something everybody wants.

So maybe one of two things:

1. We take out the default and just allow transparent bgs
2. We add the option to not only allow transparency, but if  they don't want transparency allow them to pick a default color

In the future maybe they can pick a set of colors that gets randomly picked somehow.

**Review by sk33z3r — APPROVED** [review]

**Review by sk33z3r — APPROVED** [review]

File: generate.py [hunk 1]
```diff
@@ -216,12 +216,14 @@ def main():
         for p in parts:
             composite = Image.alpha_composite(composite, p)
 
-        # Convert to RGB
-        background = Image.new(mode="RGB", size=composite.size, color=(128, 128, 128))
-        background.paste(composite, mask=composite.split()[3])  # Drop the alpha channel
+        # # Convert to RGB (uncomment section)
+        # backgroun_color = (128, 128, 128)
```
Comment (sk33z3r) [inline generate.py:220]: Potentially need to fix this, possible typo of `backgroun_color`

Comment (sk33z3r) [reply generate.py:220]: resolved in `2754d9d`

**Review by sk33z3r — APPROVED** [review]

**Pull request merged**